beacon-chain/blockchain/forkchoice/process_attestation.go

```diff
@@ -216,3 +275,8 @@ func (s *Store) updateAttVotes(
 	}
 	return nil
 }
+
+// returns true when time is divisible with slot duration / 2.
+func halfSlot() bool {
+	return uint64(time.Now().Unix())%params.BeaconConfig().SecondsPerSlot/2 == 0
```

Comment:
Would it be better to assign this with time since genesis?

Reply:
Done